crates/sui/src/ethereum_client.rs

```diff
@@ -1,5 +1,26 @@
+use anyhow::anyhow;
+use anyhow::Error;
 use clap::Subcommand;
+use move_core_types::ident_str;
+use move_core_types::identifier::IdentStr;
+use serde_json::{Number, Value};
+
+use shared_crypto::intent::Intent;
+use sui_json::SuiJsonValue;
+use sui_json_rpc_types::{SuiExecutionStatus, SuiTransactionBlockEffectsAPI}; // todo!(ide keeps deleting SuiTransactionBlockEffectsAPI)
```

Comment:
the formatter removes this this package because it is a dependency of `serialize_or_execute` macro function - so while formatting the macro is not expanded and the formatter thinks it is redundant, when it is not.